template_config.yaml

```diff
@@ -20,7 +20,23 @@ project:
 tag_operation_files: # DICOM tag anonymisation operations
     base:
         - "base-tag-operations.yaml" # Base schema
-    manufacturer_overrides: none # Manufactuer-dependendent overrides
+    manufacturer_overrides: [] # Manufactuer-dependendent overrides
+
+allowed_manufacturers:
+  - regex: "^example-manufacturer"
+    exclude_series_numbers: []
+  # For DICOM generated by Canon, Series "8000" always has bunred-in data
+  # so we always exlcude this series
+  - regex: "^canon"
+    exclude_series_numbers: ["8000"]
```

Comment:
I guess we could also add in the GE excluded series in the template as well

Reply:
oh yeah good point. I've also added `mip` to the excluded series description in the template (or should I remove it?)

Reply:
Yeah I think its a good idea, could add a comment that for PET studies this has PID